python/python/glide/async_commands/core.py

```diff
@@ -361,6 +362,39 @@ class InsertPosition(Enum):
     AFTER = "AFTER"
 
 
+def _build_sort_args(
```

Comment:
are there any conflicting combination of params? If so, need to check for that and return errors